tests/cli/runxgb.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+#run make in rabit/test to generate librabit_mock
+#update config.mk and build xgboost using mock
+export DMLC_SUBMIT_CLUSTER=local
+
+submit="python3 ../../dmlc-core/tracker/dmlc-submit"
```

Comment:
Maybe in the future we can merge this into our interface.

Reply:
sounds good. can you specify which interface you mentioned.